.eslintrc.cjs

```diff
@@ -35,6 +35,7 @@ module.exports = defineConfig({
       'error',
       { default: 'array-simple', readonly: 'generic' },
     ],
+    '@typescript-eslint/consistent-type-exports': 'error',
```

Comment:
@ST-DDT we should enable this in Faker if we did not already :eyes:

Reply:
I'll create a PR.